### PR TITLE
Update README-template.md

### PR DESCRIPTION
**Pull request by kmh4321**

### CODE REVIEW

**Comment by splovyt** [comment]
Just FYI, in the other README's we changed `fine tuning` to `fine-tuning`.

File: README-template.md [hunk 1]
```diff
@@ -167,8 +167,8 @@ To run the Flask API app in debug mode, edit `config.py` to set `DEBUG = True` u
 
 To stop the Docker container, type `CTRL` + `C` in your terminal.
 
-## Train this model
+## Train this Model on Watson Machine Learning
 
 > Remove this section if this model cannot be trained using custom data. Refer to https://github.ibm.com/CODAIT/max-model-training for details on how to enable a model for custom training.
 
-Follow [these instructions](training/README.md) to train this model using your own data.
+Whether it is fine-tuning the model with transfer learning or training from scratch on a custom dataset, retraining the model is a power feature. Please follow the steps listed under the [training readme](training/README.md) to retrain the model on [Watson Machine Learning](https://www.ibm.com/cloud/machine-learning), a deep-learning as a service (DLaaS) offering of [IBM Cloud](https://ibm.biz/Bdz2XM).
```
Comment (bdwyer2) [inline README-template.md]: The first sentence is wordy. Maybe something like this?
```suggestion
This model supports both fine tuning with transfer learning and training from scratch on a custom model. Please follow the steps listed under the [training readme](training/README.md) to retrain the model on [Watson Machine Learning](https://www.ibm.com/cloud/machine-learning), a deep-learning as a service (DLaaS) offering of [IBM Cloud](https://ibm.biz/Bdz2XM).
```

**Comment by kmh4321** [comment]
@ptitzler

**Comment by ptitzler** [comment]
We should probably provide multiple template paragraphs - one for each supported scenario. During customization one would choose the appropriate paragraph and remove the others. By providing multiple choices that don't require manual changes we'll achieve more consistency across all model README files.

Something like:

This model can be trained from scratch. Please follow the instructions in the [training README](training/README.md) to retrain the model on [Watson Machine Learning](https://www.ibm.com/cloud/machine-learning), a deep learning as a service [IBM Cloud](https://ibm.biz/Bdz2XM) offering .

This model supports fine-tuning with transfer learning and training from scratch on a custom model. Please follow the instructions in the [training README](training/README.md) to retrain the model on [Watson Machine Learning](https://www.ibm.com/cloud/machine-learning), a deep learning as a service [IBM Cloud](https://ibm.biz/Bdz2XM) offering .

**Comment by kmh4321** [comment]
@ptitzler I'll remove the DLaaS acronym. As for the other point, our training does both and the steps are common. They're not two separate functions as such.

**Pull request merged**